berkdb/rep/rep_record.c

```diff
@@ -388,6 +389,17 @@ lc_free(DB_ENV *dbenv, struct __recovery_processor *rp, LSN_COLLECTION * lc)
 	lc->nalloc = 0;
 }
 
+int normalize_rectype(u_int32_t *rectype) {
+	if (*rectype > 12000 || (*rectype > 2000 && *rectype < 10000)) {
+		assert (gbl_utxnid_log);
```

Comment:
I think remove this assert- we may need to disable utxnid logs & restart a database